cmd/manager/main.go

```diff
@@ -195,24 +198,30 @@ func main() {
 // addMetrics will create the Services and Service Monitors to allow the operator export the metrics by using
 // the Prometheus operator
 func addMetrics(ctx context.Context, cfg *rest.Config, namespace string) {
```

Comment:
Why not define this function within the metrics package?

Reply:
Along with the metricsHost + metricsPort variables

Reply:
This function is generated by `operator-sdk` here and is just creating Services and Service Monitor.  I can move it to `metrics` package